### PR TITLE
SQUID SPECIES

### PR DESCRIPTION
**Pull request by hamurlik**
image

image

Adds back the april fools squid race, ported from vmstation with changes.
Difference between humans:
are not human
can't wear shoes; are stunned by glass shards for 1.5x longer
have special footstep and attack sounds
funny as fuck
dislike fried food
funny as fuck

I tested it on a local server, works just fine

### CODE REVIEW

**Comment by AcapitalA** [comment]
> no april fools restriction
madman

File: code/datums/elements/caltrop.dm [hunk 1]
```diff
@@ -96,7 +96,10 @@
 	H.apply_damage(damage, BRUTE, picked_def_zone, wound_bonus = CANT_WOUND)
 
 	if(!(flags & CALTROP_NOSTUN)) // Won't set off the paralysis.
-		H.Paralyze(60)
+		if(H.dna.species.id == "squid")
+			H.Paralyze(90)
+
+		else H.Paralyze(60)
```
Comment (TheChosenEvilOne) [inline code/datums/elements/caltrop.dm:102]: use is_species instead, preferably add a helper here: https://github.com/The-Merchants-Guild/Merchant-Station-13/blob/master/code/__DEFINES/is_helpers.dm#L59

File: code/modules/mob/living/carbon/human/species_types/squidpeople.dm [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/datum/species/squid
+	name = "Squidperson"
+	id = "squid"
+	say_mod = "spits out"
+	default_color = "b8dfda"
+	species_traits = list(MUTCOLORS,EYECOLOR,HAS_FLESH,HAS_BONE,LIPS)
+	inherent_traits = list(TRAIT_ADVANCEDTOOLUSER,TRAIT_CAN_STRIP,TRAIT_CAN_USE_FLIGHT_POTION)
+	payday_modifier = 0.75
+//	default_features = list("mcolor" = "FFF") // bald
```
Comment (TheChosenEvilOne) [inline code/modules/mob/living/carbon/human/species_types/squidpeople.dm:9]: remove commented out code

File: code/modules/mob/living/carbon/human/species_types/squidpeople.dm [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	no_equip = list(ITEM_SLOT_FEET)
+	skinned_type = /obj/item/stack/sheet/animalhide/human
+	ass_image = 'icons/ass/asssquid.png'
+	special_step_sounds = list('sound/effects/footstep/squidwalk1.ogg', 'sound/effects/footstep/squidwalk2.ogg' )
+	attack_verb = "slap"
+	attack_sound = 'sound/weapons/whip.ogg'
+	miss_sound = 'sound/weapons/whipmiss.ogg'
+	grab_sound = 'sound/weapons/whipgrab.ogg'
+	disliked_food = FRIED
+//	nojumpsuit = TRUE
```
Comment (TheChosenEvilOne) [inline code/modules/mob/living/carbon/human/species_types/squidpeople.dm:22]: ditto

File: code/modules/mob/living/carbon/human/species_types/squidpeople.dm [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+/mob/living/carbon/human/species/squid
+	race = /datum/species/squid
+
+//	/datum/species/squid/qualifies_for_rank(rank, list/features)
+//		return TRUE
```
Comment (TheChosenEvilOne) [inline code/modules/mob/living/carbon/human/species_types/squidpeople.dm:28]: ditto

File: code/modules/mob/living/carbon/human/species_types/squidpeople.dm [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+/proc/random_unique_squid_name(attempts_to_find_unique_name=10, genderToFind)
+	for(var/i in 1 to attempts_to_find_unique_name)
+		. = capitalize(squid_name(genderToFind))
+		if(!findname(.))
+			break
+
+/datum/mood_event/squid //april 1st only
+	description = "<span class='nicegreen'>Spongebob isn't on this station with me.</span>\n" //Used for syndies, nukeops etc so they can focus on their goals
+	mood_change = 0
+	hidden = FALSE
```
Comment (TheChosenEvilOne) [inline code/modules/mob/living/carbon/human/species_types/squidpeople.dm:50]: we don't have mood enabled, just remove this.

File: config/game_options.txt [hunk 6]
```diff
@@ -420,6 +420,7 @@ ROUNDSTART_RACES lizard
 ROUNDSTART_RACES moth
 ROUNDSTART_RACES plasmaman
 #ROUNDSTART_RACES shadow
+ROUNDSTART_RACES squid
```
Comment (TheChosenEvilOne) [inline config/game_options.txt:423]: config changes here don't affect the server

**Review by TheChosenEvilOne — COMMENTED** [review]
I don't really like the idea of adding a new species, but whatever

**Comment by AcapitalA** [comment]
> I don't really like the idea of adding a new species, but whatever

Same tbh, i'd be alright with it if it was april fools only.

**Comment by trollertroll** [comment]
> I don't really like the idea of adding a new species, but whatever

It isn't fetish content, it's very funny
You are fine with farts so I don't see anything wrong with squids

But anyway, I did the requested changes, except the is_species, because it gives me an unknown variable error.

**Comment by QuakeIV** [comment]
i was extremely against this until i realised it was squidward

**Comment by QuakeIV** [comment]
image
can it look more like squidward

**Comment by QuakeIV** [comment]
eh more seriously though duno if we are going to merge this, we arent that big on nonhuman species in general i think

**Comment by eLeCtrOssSnake** [comment]
I don't like new species unless it's catgirls.

**Comment by nicholast013** [comment]
fuck off

**Comment by QuakeIV** [comment]
the squid species has been expelled 1 gorillion times and yet it is always the humans fault?

**Pull request closed without merging**